docs/make.jl

```diff
@@ -0,0 +1,25 @@
+push!(LOAD_PATH,"../src/")
+
+using Documenter
+using MCMCChains
+
+name = "MCMCChains.jl"
+
+makedocs(
+    sitename = name,
```

Comment:
I think `name` should be removed - it is too generic and we will never change the name anyway:
```suggestion
makedocs(
    sitename = "MCMCChains.jl",
```